internal/apex/ctlserver.go

```diff
@@ -0,0 +1,84 @@
+package apex
+
+import (
+	"fmt"
+	"net"
+	"net/rpc"
+	"net/rpc/jsonrpc"
+	"os"
+)
+
+// TODO make this path configurable
+const UnixSocketPath = "/var/run/apex.sock"
+
+// Assumptions made in here:
+//  - ax.logger is thread-safe
```

Comment:
That is a fine assumption. Yes Zap is thread-safe.

Reply:
Thanks :)